src/servers/src/opentsdb.rs

```diff
@@ -126,3 +130,37 @@ impl Server for OpentsdbServer {
         OPENTSDB_SERVER
     }
 }
+
+pub fn data_point_to_grpc_row_insert_requests(
+    data_points: &[DataPoint],
+) -> Result<(RowInsertRequests, usize)> {
+    let mut multi_table_data = MultiTableData::new();
+
+    for data_point in data_points {
+        let table_name = data_point.metric();
+        let tags = data_point.tags().clone();
+        let value = data_point.value();
+        let timestamp = data_point.ts_millis();
+        let num_columns = tags.len() + 1;
+
+        let table_data = multi_table_data.get_or_default_table_data(table_name, num_columns, 0);
```

Comment:
There's at least one row for each table:

```suggestion
        let table_data = multi_table_data.get_or_default_table_data(table_name, num_columns, 1);
```